src/lib/snyk-to-html.ts

```diff
@@ -20,6 +24,27 @@ function readFile(filePath: string, encoding: string): Promise<string> {
   });
 }
 
+function handleInvalidJson(reason: any) {
+  if (!reason.isInvalidJson) {
+    reason.message = 'Cannot run `snyk-to-html`. Make sure you are providing the correct parameters.';
```

Comment:
could you change this to `Error running `snyk-to-html`. Please check you are providing the correct parameters. Is the issue persists contact support@snyk.io`

Reply:
sure